pages/common/docker-login.md

```diff
@@ -0,0 +1,20 @@
+# docker login
+
+> Log into a docker registry.
+> More information: <https://docs.docker.com/engine/reference/commandline/login/>.
+
+- Log into registry (user will be prompted for a username and password):
+
+`docker login`
+
+- Log into registry with a specific username (user will be prompted for a password):
+
+`docker login --username {{username}}`
+
+- Log into registry with username and password:
+
+`docker login --username {{username}} --password {{password}} {{server}}`
+
+- Log into registry with password from stdin:
+
+` echo "{{password}}" | docker login --username {{username}} --password-stdin`
```

Comment:
I think we shouldn't encourage the user to directly write a password into the prompt. How about this instead:
```suggestion
` cat {{password_file}} | docker login --username {{username}} --password-stdin`
```

Reply:
there was a long discussion about that: https://github.com/tldr-pages/tldr/pull/6137#discussion_r653914522

Reply:
We are degrading the usage because Windows can't into posix.